floss/main.py

```diff
@@ -528,6 +529,10 @@ def main(argv=None) -> int:
     # 3. tight strings
     # 4. decoded strings
 
+    if is_go_bin(sample=sample):
+        logger.warning("GO Binary Detected, GOLang binaries are not supported yet. Results may be inaccurate.")
+        logger.warning("GO: Proceeding with analysis may take a long time.")
```

Comment:
```suggestion
        logger.warning("Go Binary Detected, Go binaries are not supported yet. Results may be inaccurate.")
        logger.warning("Go: Proceeding with analysis may take a long time.")
```

The language name is "Go", which can be difficult to Google for, since its such as short name, hence the convention that people sometimes use "golang". but we should use "Go" here because its the correct name.